pages/attacks/SQL_Injection.md

```diff
@@ -43,7 +41,7 @@ See the [OWASP Code Review Guide](https://owasp.org/www-project-code-review-guid
 
 ### How to Test for SQL Injection Vulnerabilities
 
-See the [OWASP Testing Guide](https://owasp.org/www-project-web-security-testing-guide/) for information on testing for SQL Injection vulnerabilities.
+See the [OWASP Testing Guide](https://github.com/OWASP/wstg/blob/master/document/4-Web_Application_Security_Testing/07-Input_Validation_Testing/05-Testing_for_SQL_Injection.md) for information on testing for SQL Injection vulnerabilities.
```

Comment:
This shouldn't be changed. It could link to the v4.2 content published on the site but it's still probably best to just leave it as it was that way users can easily check latest, stable, or versioned WSTG content.

Reply:
That makes sense, thank you.  Sorry about that.  By the way, I'm glad to see that you've added a section on threat modeling.